app/client/src/components/shared/WaterbodyInfo/index.js

```diff
@@ -1250,11 +1250,15 @@ function UsgsStreamgagesContent({ feature }: { feature: Object }) {
 
   const sortedPrimaryMeasurements = [...primaryMeasurements]
     .sort((a, b) => a.parameterOrder - b.parameterOrder)
-    .map((data) => <UsgsStreamgageParameter url={locationUrl} data={data} />);
+    .map((data, index) => (
+      <UsgsStreamgageParameter url={locationUrl} data={data} key={index} />
+    ));
 
   const sortedSecondaryMeasurements = [...secondaryMeasurements]
     .sort((a, b) => a.parameterName.localeCompare(b.parameterName))
-    .map((data) => <UsgsStreamgageParameter url={locationUrl} data={data} />);
+    .map((data, index) => (
+      <UsgsStreamgageParameter url={locationUrl} data={data} key={index} />
+    ));
```

Comment:
Instead of using the index as the key (which won't be unique if there are secondary measurements), I'm changing this to use the parameter code (which is unique). I was doing that before, but I passed it in the wrong place (inside the component's rendered output, instead of the component itself that's being mapped), so thanks for flagging this!

Reply:
Good catch!